server/README.md

```diff
@@ -12,6 +12,21 @@ pre-requiste: node.js is already installed on your machine
 
 ## Active endpoints
 * ANY - https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/home
+* GET - https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/viewProfile
```

Comment:
Would it be possible to rename to something more [RESTful](https://restfulapi.net/resource-naming/)? It seems that from the documentation that something like `"/profile/{id}"` is more RESTful and best practice.

Reply:
This is a believe limitation of API gateway (or a knowledge gap on my part). This API is hidden from the clients anyways - All lambda APIs are called by react (fetch())